### PR TITLE
Fix 11273 remove exception log is output in console

### PR DESCRIPTION
**Pull request by sksDonni**
### Fix [banyandb web-ui console error logs](https://github.com/apache/skywalking/issues/11273)
Experiencing exception logs in the console of the banyandb dashboard when we switch between two read tabs. 

- [ ] Add a unit test to verify that the fix works.

- [X] Explain briefly why the bug exists and how to fix it.
The issue was caused due to the use of `JSON.parse(JSON.stringify())` to clone the route object. This method of cloning objects cannot handle circular references, leading to an exception in the console.

- [ ] If this is non-trivial feature, paste the links/URLs to the design doc.
- [ ] Update the documentation to include this new feature.
- [ ] Tests(including UT, IT, E2E) are added to verify the new feature.
- [X] If it's UI related, attach the screenshots below.

https://github.com/apache/skywalking-banyandb/assets/53624363/b62bba6e-2678-4e8a-9b3e-4436cd0f5649

- [X] If this pull request closes/resolves/fixes an existing issue, replace the issue number. Fixes apache/skywalking#11273.
- [ ] Update the [`CHANGES` log](https://github.com/apache/skywalking-banyandb/blob/main/CHANGES.md).

### CODE REVIEW

File: ui/src/components/Read/index.vue [hunk 1]
```diff
@@ -184,8 +184,10 @@ watch(() => data.code, () => {
     if (index >= 0) {
         data.codeStorage[index].params.code = data.code
     } else {
+
         route.params.code = data.code
-        data.codeStorage.push(JSON.parse(JSON.stringify(route)))
+        const clonedObject = Object.assign({}, route);
```
Comment (hanahmily) [inline ui/src/components/Read/index.vue:189]: After researching, I found that this code only creates a shallow copy, not a deep copy, which could lead to potential issues. Can you take a look, @Fine0830?

Comment (hanahmily) [reply ui/src/components/Read/index.vue:189]: https://github.com/WebReflection/flatted might help with this issue.

Comment (Fine0830) [reply ui/src/components/Read/index.vue:189]: The `Object. assign` can only deeply copy the first layer of json.

Comment (wfxSuchith) [reply ui/src/components/Read/index.vue:189]: sure @hanahmily and @Fine0830, will look into this and update the PR accordingly.

Comment (sksDonni) [reply ui/src/components/Read/index.vue:189]: Hi @Fine0830 @hanahmily, tried using structuredClone(). Am encountering some issue similar to this [here](https://stackoverflow.com/questions/72632173/unable-to-use-structuredclone-on-value-of-ref-variable). Will try to manually loop and clone each child recursively.

Comment (Fine0830) [reply ui/src/components/Read/index.vue:189]: Could you use `toRow` before using `structuredClone`?

Comment (hanahmily) [reply ui/src/components/Read/index.vue:189]: Did you try using "toRaw" as suggested in the link you shared? This solution may help resolve the problem you encountered.

Comment (Fine0830) [reply ui/src/components/Read/index.vue:189]: Probably, it is enough to use `toRaw` here. Transforming a responsive object generated with reactive into a non responsive regular object, then assign a value to the new variable. All operations on this object will not cause page updates.

Comment (sksDonni) [reply ui/src/components/Read/index.vue:189]: @Fine0830 @hanahmily, Using toRaw is still causing the issue to persist. Will see what else can be done. Will revert to a recursive solution if nothing works out.

Comment (Fine0830) [reply ui/src/components/Read/index.vue:189]: How do you use `toRaw`, Only toRaw(route)? If so, it is not enough. All data in the `route` need to be converted.

Comment (sksDonni) [reply ui/src/components/Read/index.vue:189]: Yeah, tried using toRaw(route), toRaw(route.data) and toRaw(route.value). Nothing seemed to work for me

Comment (hanahmily) [reply ui/src/components/Read/index.vue:189]: You could try JSON.parse (JSON. stringify()). If it doesn't work, writing a custom function might be the only solution.

**Comment by sksDonni** [comment]
Had closed this pull request accidentally. Requesting second review.

**Comment by Fine0830** [comment]
While debugging this code, I noticed that it wasn't executing in development mode and encountered errors in production mode, without affecting the page interaction. As a result, I concluded that this segment of code was invalid and proceeded to delete it.

**Comment by wu-sheng** [comment]
@Fine0830 Will you use another PR to remove the codes?

**Comment by Fine0830** [comment]
I have removed the codes in https://github.com/apache/skywalking-banyandb/pull/408

**Pull request closed without merging**